hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/TokenFeature.java

```diff
@@ -731,6 +733,42 @@ public void verifyMirrorNftTransfer(AccountClient.AccountNameEnum accountName) {
         verifyNftTransactions(tokenId, serialNumber);
     }
 
+    @Then("I airdrop {int} tokens to {account}")
+    public void airdropTokens(int amount, AccountNameEnum accountName) {
+        var receiver = accountClient.getAccount(accountName);
+        var sender = accountClient.getSdkClient().getExpandedOperatorAccountId();
+        networkTransactionResponse = airdropFungibleTokens(tokenId, amount, sender, receiver);
```

Comment:
airdropFungibleTokens() helper can also be removed.

```suggestion
        verify(tokenClient.executeFungibleTokenAirdrop(tokenId, sender, receiver.getAccountId(), amount));
```

Reply:
Done